### DISCUSSION TITLE
Dark theme

### DISCUSSION THREAD

**Discussion by homeplayer2** [thelastoutpostworkshop/gpio_viewer#147] [Ideas]
Would love to see a dark them, (I really dont like white backgrounds,

i have edit gpio_viewer.h and add 3 lines after line 415,
        html += "<script type='text/javascript'>";
        html += "document.body.style.backgroundColor = '#282424'"; // set background color
        html += "</script>";
        
Maybe i am just lost, but i see theme vars in the code, but cant find them to edit?

![Screenshot from 2024-12-24 11-15-38](https://github.com/user-attachments/assets/b9c9c56b-5cd6-4971-9fc3-395de2423e7d)

**Comment by thelastoutpostworkshop** [comment]
Very good idea, thanks

**Comment by thelastoutpostworkshop** [comment]
I released a new version of the web application (2.1.6) with a new menu to toggle between dark and light theme.
![image](https://github.com/user-attachments/assets/b21d40df-9705-4d37-831f-2eee6dc86791)
